lib/data_cycle/batch_update_logging.rb

```diff
@@ -59,6 +59,7 @@ def log_end_of_update(message)
     log_message 'Update finished'
     total_time = distance_of_time_in_words(@start_time, @end_time)
     Rails.logger.info "#{message} Time: #{total_time}."
+    UpdateLog.log_update(@end_time.to_datetime) unless message == 'Daily update finished.'
```

Comment:
Relying on that message string is fragile... especially with matching on an `unless` condition. If a DailyUpdate fails partway through, it still hits this code but with the message 'Daily update failed'.

Maybe instead do `if self.class == ConstantUpdate`.

Reply:
Definitely. I was certainly not sure about using the message. I just pushed your proposal asi I think it's a better solution, indeed.

Reply:
Cool. Assuming the tests pass — I *think* that suggestion will work, but not entirely sure — I'll merge this. I think later we'll want to log more info, including info about all types of updates (we also have SurveyUpdate in addition to Constant and Daily), but for an initial version, this looks solid.